tests/unit/test_preprocessing.py

```diff
@@ -8,8 +8,19 @@ class TestPreprocessing(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        cls.v4e4 = graph_vertex4edge4()
-        cls.v5e4loop = graph_vertex5edge4loop()
+        #         Create a graph with structure:
+        #
+        #         1 - 2 - 3
+        #              \ /
+        #               4
+        cls.v4e4 = create_graph_helper([(1, 2), (2, 3), (2, 4), (3, 4)])
+
```

Comment:
Kunnen we deze niet globaal maken zoals volgens mij @erikhuizinga ook al ergens gedaan heeft? Want dan hoeft dit dus niet twee keer voor dezelfde graaf

Reply:
Yup, namelijk op de volgende plekken:

https://github.com/MinThaMie/graph_isomorphism/blob/master/tests/__init__.py#L5-L14

en in je `setUp` / `setUpClass`:

https://github.com/MinThaMie/graph_isomorphism/blob/master/tests/__init__.py#L17

Reply:
En zoals ik al zei: daar staan mogelijk wat grafen die jij ook maakt, dus kun je hergebruiken.